packages/firestore/src/local/indexeddb_remote_document_cache.ts

```diff
@@ -42,7 +42,7 @@ import { SimpleDbStore } from './simple_db';
 
 export class IndexedDbRemoteDocumentCache implements RemoteDocumentCache {
   /** The last id read by `getNewDocumentChanges()`. */
-  private lastReturnedDocumentChangesId = 0;
+  private _lastProcessedDocumentChangesId = 0;
```

Comment:
Can we be consistent with 'ChangeId' or 'ChangesId' ?  I think I prefer the former.

Reply:
I renamed all "changesId" to "changeId".